client/controller/api/apps.go

```diff
@@ -6,7 +6,7 @@ type App struct {
 	ID      string `json:"id"`
 	Owner   string `json:"owner"`
 	Updated string `json:"updated"`
-	URL     string `json:"url"`
+	URL     string
```

Comment:
@helgi can you either name this field with a struct tag (basically, add the `json:"url"` back) or add a `json:"-"` if it's supposed to be ignored?